htdocs/salaries/card.php

```diff
@@ -662,9 +669,9 @@
 	print '<script>';
 	print '$( document ).ready(function() {';
 		print '$("#updateAmountWithLastSalary").on("click", function updateAmountWithLastSalary() {
+					console.log("We click on link to autofill salary amount url="+url);
```

Comment:
Why moving this line up ?
The line is nod before the line var url = ...
but this line include the value url into its log.

Reply:
Don't know why.
This is a probably mistake.